libs/login/WordPressLoginFlow/src/main/res/layout/toolbar_login.xml

```diff
@@ -17,9 +17,9 @@
             android:layout_height="24dp"
             android:layout_gravity="center"
             android:contentDescription="@null"
-            android:tint="?attr/colorControlNormal"
             android:visibility="gone"
-            app:srcCompat="@drawable/login_toolbar_icon" />
+            app:srcCompat="@drawable/login_toolbar_icon"
+            app:tint="?attr/colorControlNormal" />
```

Comment:
This is just a change from `android:tint` to `app:tint` I've manually made to fix the lint error in the library.